src/Hashable.php

```diff
@@ -20,15 +20,15 @@ interface Hashable
      *
      * @return mixed
      */
-    function hash();
+    public function hash();
```

Comment:
To my understanding, interfaces usually do not include the visibility, but I do not have a strong opinion here. I'm happy with this change. :+1: